src/encoded/static/components/browse/components/file-tables.js

```diff
@@ -49,6 +49,7 @@ export class RawFilesStackedTable extends React.Component {
     }
 
     /**
+     * @deprecated ?
```

Comment:
We're currently/tentatively planning to remove remainder of expSetFilters stuff and simply read them off of the current href. Running through Filters.hrefToFilters for where/when needed, most likely.